scripts/split.sh

```diff
@@ -61,17 +61,21 @@ then
   mkdir -vp "$TESTDIR" "$TRAINDIR"
 fi
 
-let n="$(ls $TESTDIR | wc -l) + $(ls $TRAINDIR | wc -l)"
+let n_test="$(ls $TESTDIR | wc -l)"
+let n_train="$(ls $TRAINDIR | wc -l)"
+let n="$n_test + $n_train"
 let overhead="$WINSIZE / 10"
-let max="$WINSIZE + $overhead + 100"
+let max="$WINSIZE + $overhead + 200"
+max_train=$(echo "scale=1;($TRAINPCT / 100) * $max" | bc | cut -d'.' -f1)
+max_test=$(echo "scale=1;(1 - $TRAINPCT / 100) * $max" | bc | cut -d'.' -f1)
```

Comment:
Couldn't this just be `let max_test="1 - $max_train"` ?

Reply:
it would be max - max_train - but I'll leave that for a potential follow up.